query-node/mappings/src/content/nft.ts

```diff
@@ -952,24 +958,19 @@ export async function contentNft_NftSlingedBackToTheOriginalArtist({
 }: EventContext & StoreContext): Promise<void> {
   // common event processing
 
-  const [videoId, ownerId] = new Content.NftBoughtEvent(event).params
+  const [videoId, ownerId] = new Content.NftSlingedBackToTheOriginalArtistEvent(event).params
```

Comment:
This change is ok, but `ownerId` is of type `ContentActor` now -> we need to reflect that change of definition in `NftSlingedBackToTheOriginalArtistEvent` in schema and on lane `owner: new Membership({ id: ownerId.toString() }),`